client/src/components/icons/cloud-off.tsx

```diff
@@ -0,0 +1,35 @@
+import { SVGProps } from "react";
```

Comment:
I think you can replace this icon with one of these: https://react-icons.github.io/react-icons/search/#q=cloudoff

Reply:
Why? We currently don't use this package